README.md

```diff
@@ -13,6 +13,24 @@ This extension modifies the Saved Replies for issues or pull requests on any rep
 
 Get it on [Chrome Web Store > Refined Saved Replies](https://chrome.google.com/webstore/detail/refined-saved-replies/ngcinicnlicdndmpcfjjifononfcceih)! ✨
 
+Install it from the [Firefox Add-ons > Refined Saved Replies](#TBD) page! 🦊
+
```

Comment:
[Proofreading] Just spiffying up some formatting... thoughts?

```suggestion
Get it on...

* [Chrome Web Store > Refined Saved Replies](https://chrome.google.com/webstore/detail/refined-saved-replies/ngcinicnlicdndmpcfjjifononfcceih)! ✨
* [Firefox Add-ons > Refined Saved Replies](#TBD)! 🦊
```

Reply:
For the `#TBD`, I think it's fine to make this not a link for now, and instead say _(pending approval)_ or something like that.